### DISCUSSION TITLE
Background update of store app can break other app using font shipped with it

### DISCUSSION THREAD

**Discussion by huoyaoyuan** [microsoft/WindowsAppSDK#384] [Q&A]
I'm setting main font of my Visual Studio to Cascadia Code, which is shipped with Windows Terminal by store.

If Windows Terminal is updated with VS open, the text rendering of VS will become strange.

The ideal behavior is that existing process will get a temporary copy of the font, handled by either system or application framework.

Requiring reboot or totally blocking update here is not ideal.

**Comment by stevewri** [comment]
@niklasb-ms - not sure if this is just a fonts issue or a fonts and deployment issue.
